plugins/services/src/js/components/modals/CreateServiceModal.js

```diff
@@ -491,6 +497,32 @@ class CreateServiceModal extends Component {
     this.setState({ isPending: true });
   }
 
+  setShouldExpand() {
```

Comment:
this is a very broad name for `Advanced section of the General Config Tab`. Having `shouldExpand` in this context implies that the method controls whether the whole `CreateServiceModal` should expand.

Reply:
Good point, I will change the name of this function.